app/controllers/api/v1/actions_controller.rb

```diff
@@ -43,7 +43,7 @@ def rbac_scope(relation)
 
         # Only approver can reach here
         action_ids = approver_id_list(relation.model.table_name)
-        Rails.logger.info("approver scope for actions: #{action_ids}")
+        Rails.logger.info { "Number of approver scope for actions: #{action_ids.count}" }
```

Comment:
either `debug { }` or `info()`
If the actual IDs are needed for debugging, you can still log all `action_ids` with `debug {}`.